### PR TITLE
Ruff: Add and fix PTH122

### PR DESCRIPTION
**Pull request by manuel-sommer**

### CODE REVIEW

File: dojo/utils.py [hunk 1]
```diff
@@ -1381,7 +1381,9 @@ def get_page_items_and_count(request, items, page_size, prefix="", do_count=True
 
 
 def handle_uploaded_threat(f, eng):
-    _name, extension = os.path.splitext(f.name)
+    path = Path(f.name)
+    _name = path.parent / path.stem
```
Comment (cneill) [inline dojo/utils.py:1385]: Looks like this can be removed since it's unused

```suggestion

```

File: dojo/utils.py [hunk 2]
```diff
@@ -1395,7 +1397,9 @@ def handle_uploaded_threat(f, eng):
 
 
 def handle_uploaded_selenium(f, cred):
-    _name, extension = os.path.splitext(f.name)
+    path = Path(f.name)
+    _name = path.parent / path.stem
```
Comment (cneill) [inline dojo/utils.py:1401]:
```suggestion

```

**Review by cneill — COMMENTED** [review]
Just some quick cleanup, otherwise looks good

**Review by cneill — APPROVED** [review]

**Review by mtesauro — APPROVED** [review]
Approved

**Review by Maffooch — APPROVED** [review]

**Review by grendel513 — APPROVED** [review]

**Pull request merged**